Assets/MRTK/Core/Providers/Hands/ArticulatedHandDefinition.cs

```diff
@@ -92,7 +92,8 @@ public float ExitPinchDistance
             new MixedRealityInteractionMapping(1, "Spatial Grip", AxisType.SixDof, DeviceInputType.SpatialGrip),
             new MixedRealityInteractionMapping(2, "Select", AxisType.Digital, DeviceInputType.Select),
             new MixedRealityInteractionMapping(3, "Grab", AxisType.SingleAxis, DeviceInputType.TriggerPress),
-            new MixedRealityInteractionMapping(4, "Index Finger Pose", AxisType.SixDof, DeviceInputType.IndexFinger)
+            new MixedRealityInteractionMapping(4, "Index Finger Pose", AxisType.SixDof, DeviceInputType.IndexFinger),
+            new MixedRealityInteractionMapping(5, "Teleport Pose", AxisType.DualAxis, DeviceInputType.ThumbStick)
```

Comment:
What are the two axes here? Is it pitch and yaw?

Reply:
The whole process with creating mixed reality input mappings was a bit weird. Basically, an input mapping creates an "action" (in this case, Teleport Pose) and associates it with a DeviceInputType, which in our case, is the thumbstick. This mirrors how we currently use the control stick to initiate a teleport when using controllers on devices. The other existing input mappings operate under the same logic (the Select action is associated with the "select" DeviceInput Type, the Grab action is associated with pressing the trigger)

In accordance with this framework, we have to then list the Thumbstick as the proper axis type, which is a DualAxis.

Now, these interaction mappings mostly exist to establish proper relationships between controller input and the actions we want to do (I think this is the correct interpretation, @keveleigh if I messed anything up). It is then up to the various implementations that raise events (ie keyboard input simulation, the mechanisms that recognize the pose a hand makes) to then raise the appropriate action through it's interaction mapping (you'll see in the ArticulatedHandDefinition code that the definition raises a teleport input with RaiseInputPositionChanged)

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/520b5d730fb3d067a760f50585c5ee39d49567ae/Assets/MRTK/Core/Providers/Hands/ArticulatedHandDefinition.cs#L288